projects/GKCore/GKCore/Controllers/PersonEditDlgController.cs

```diff
@@ -340,16 +340,13 @@ private void SetTarget(GDMIndividualRecord value)
                 if (fTarget != null) {
                     INamesTable namesTable = AppHost.NamesTable;
 
-                    var parts = GKUtils.GetNameParts(fBase.Context.Tree, fTarget);
+                    var parts = GKUtils.GetNameParts(fBase.Context.Tree, fTarget, false);

```

Comment:
last argument is `formatted`. Default is `true` and it adds parenthesis for the maiden surname, so father of a girl will receive the surname in parenthesis. But we, obviously, do not want it.